crates/router/src/core/payments/operations/payment_reject.rs

```diff
@@ -264,7 +265,13 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, PaymentsCancelRequest> for Payme
             )
             .await
             .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
-
+        
+        req_state
+            .event_context
+            .event(AuditEvent::new(AuditEventType::PaymentReject))
+            .with(payment_data.to_event())
+            .emit();
```

Comment:
Can you send `error_code` and `error_message` as well to be logged here
You can take reference from this PR: [https://github.com/juspay/hyperswitch/pull/4913/files](https://github.com/juspay/hyperswitch/pull/4913/files)